modules/mono/csharp_script.cpp

```diff
@@ -2035,7 +2035,7 @@ void GD_CLR_STDCALL CSharpScript::_add_property_info_list_callback(CSharpScript
 #ifdef TOOLS_ENABLED
 	p_script->exported_members_cache.push_back(PropertyInfo(
 			Variant::NIL, p_script->type_info.class_name, PROPERTY_HINT_NONE,
-			p_script->get_path(), PROPERTY_USAGE_CATEGORY));
+			p_script->get_path().is_empty() ? p_script->get_class_name() : StringName(p_script->get_path()), PROPERTY_USAGE_CATEGORY));
```

Comment:
When the Godot editor first starts, this area is empty, which causes the default TSCN file's hints to be ineffective. After rebuilding the project, this area will no longer be empty.

Reply:
Do you mean that before the C# project is built for the first time, scripts have no path? If so, that may be expected since those scripts are invalid. The C# project needs to be built first so scripts can be valid, and if they project hasn't been built you'd have no documentation anyway.